docs/get-started.md

```diff
@@ -42,4 +42,4 @@ These `--git` arguments are then used to set up and configure [flux](https://www
 
 We will rely on the user installing [fluxctl](https://docs.fluxcd.io/en/latest/references/fluxctl.html#installing-fluxctl) to interact with flux directly instead of trying to replicate the functionality within `wksctl`
 
-To see a more detailed example combining Wksctl, [GitOps](https://www.weave.works/technologies/gitops/), [Ignite](https://ignite.readthedocs.io/en/stable/) also know as FireKube see [Firekube](examples/footloose/README.md#firekube-gitops)
+To see a more detailed example combining Wksctl, [GitOps](https://www.weave.works/technologies/gitops/), [Ignite](https://ignite.readthedocs.io/en/stable/) also know as FireKube see [Firekube](wks-and-footloose.md).
```

Comment:
Don't we want to deep link to `wks-and-footloose.md#run-a-kubernetes-cluster-with-firekube` instead?

Reply:
Sure.

Or do we want to point to https://github.com/weaveworks/wks-quickstart-firekube instead?

Reply:
> Or do we want to point to https://github.com/weaveworks/wks-quickstart-firekube instead?

I'm not sure, the sentence starts with _To see a more detailed example..._ so I guess it should point directly to an example if possible :)

Reply:
I didn't write the sentence.

I just wonder if we want to maintain the example doc and the full demo separately. Seems a bit much to me?

Reply:
I didn't write that sentence either :)

I would remove the firekube steps here and point at https://github.com/weaveworks/wks-quickstart-firekube which is more streamlined.

Reply:
Sweet. I'll do that!